endpoint/data_backends/__init__.py

```diff
@@ -0,0 +1,9 @@
+import importlib
+
+
+class DataLayer(object):
+    api = None
```

Comment:
What do you think of this?

Can the ‘api’ be encloses by the DataLayer?
Such that method calls on the ‘api’ are internalized?
Like ‘DataLayer.init(...)’ is a wrapper for ‘DataLayer.api.init(...)’.

Reply:
Yup yup ... great suggestion. Thanks, will do this!

Reply:
I like that! 👌👍. We can extend the object if necessary.